web/modules/custom/portland/modules/portland_location_picker/src/Element/PortlandLocationPicker.php

```diff
@@ -208,28 +208,33 @@ public static function getCompositeElements(array $element) {
       '#description_display' => 'before',
     ];
     $element['location_asset_id'] = [
-      '#type' => 'hidden',
+      '#type' => 'textfield',
       '#title' => t('Asset ID'),
       '#title_display' => 'invisible',
       '#id' => 'location_asset_id',
+      '#attributes' => ['class' => ['location-asset-id', 'visually-hidden']],
```

Comment:
`visually-hidden` means screen readers would still read these fields. (in older Bootstrap versions it actually used to be called `sr-only`, which I personally think is a better name LOL)

And I just noticed this was already an issue for the Location lat/lon and x/y. :/

The correct class to use here would be `d-none`. *But,* I tested and we can use the proper `hidden` field type by setting the ID directly as an HTML attribute:

 ```php
 '#attributes' => ['id' => 'location_asset_id'],
 ```
 
 Could you change these back to hidden using that fix, and also fix the other location fields?

Reply:
Requested changes have been made.